eox_tenant/models.py

```diff
@@ -108,6 +108,29 @@ def get_value_for_org(cls, org, val_name):
 
         return None
 
+    @classmethod
```

Comment:
@luismorenolopera I have a suggestion related to code organization. I notice the code below is similar to the one you added on the **TenantConfig** model, what changes is the model fields to use in the query. In this order of ideas, we can:

- Move this method to a model manager (named **filter_on_json_fields** or something much better)
- This method should be able to create the query for a variable number of fields
- The fields to use in the query can be defined via a model attribute, a model method, a manager method or accept a list of fields as a param on the **filter_on_json_fields** method
- Make involved models use the manager

This could be even a mixin to extend a manager. Does it make sense?